services/aur/aur.service.js

```diff
@@ -108,7 +108,7 @@ class AurVotes extends BaseAurService {
 
 class AurVersion extends BaseAurService {
   static category = 'version'
-
+  static _cacheLength = 3600
```

Comment:
Open to alternative suggestions, but I'm very intentionally swinging fairly big here. Of the >18k AUR requests we've had over the last 24 hours, nearly 16k of those were for the version badges.

I realize that version badges tend to be more sensitive to recency needs, but I think we'll need to be aggressive here if we're to have a chance of staying within AURs 4k/day limit